docs/configuration-reference.md

```diff
@@ -3023,3 +3026,94 @@ Full reference of options:
   # Parse SignalFX style tags.
   [parse_signalfx_tags: <bool> | default = true]
 ```
+
+### consul_exporter_config
+
+The `statsd_exporter_config` block configures the `statsd_exporter`
```

Comment:
statsd_exporter stuff here looks like copy pasta.

Reply:
That's because it is! 🙃

Reply:
Fixed. Thanks!